db/migrations/2023082817320000-fill-deliverables-with-pullRequests-table.js

```diff
@@ -0,0 +1,94 @@
+const { Octokit } = require("octokit");
+const { getAllFromTable } = require("../../helpers/db/rawQueries");
+const { ipfsAdd } = require("../../helpers/db/ipfsAdd");
+
+const { NEXT_GH_TOKEN, NEXT_PUBLIC_HOME_URL } = process.env;
+
+async function up(queryInterface, Sequelize) {
+  const pullRequests = await getAllFromTable(queryInterface, "pull_requests");
+  const issues = await getAllFromTable(queryInterface, "issues");
+
+  const openIssues = issues?.filter(
+    (issue) => issue.state !== "pending" && !!issue.issueId
+  );
+
+  if (!pullRequests?.length) return;
+
+  const repositories = await getAllFromTable(queryInterface, "repositories");
```

Comment:
this is too expensive, and the find afterwards even more so. produce a sql statement for what you want instead:

```sql
select c."chainShortName", n."name", i."githubId", i."contractId", i."id", p
  from chains c, issues i, pull_requests p, networks n
  where i.state != 'pending' and i."issueId" is not null
```

This will give what you want